iroh-net/src/magicsock.rs

```diff
@@ -1548,46 +1537,6 @@ impl Handle {
     }
 }
 
-/// Stream returning local endpoints as they change.
```

Comment:
This code is only moved, not changed.

Reply:
no longer true, the new code yields `BTreeSet<DirectAddr>` instead of `Vec<DirectAddr>`